marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/gradle/GradleAndroidTestBundle.kt

```diff
@@ -3,10 +3,15 @@ package com.malinskiy.marathon.gradle
 import com.android.build.api.variant.BuiltArtifactsLoader
 import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.Nested
+import java.io.Serializable
 
 data class GradleAndroidTestBundle(
-    val apkFolder: DirectoryProperty? = null,
-    val artifactLoader: Property<BuiltArtifactsLoader>? = null,
-    val testApkFolder: DirectoryProperty,
-    val testArtifactLoader: Property<BuiltArtifactsLoader>,
-)
+    @InputDirectory val apkFolder: DirectoryProperty? = null,
```

Comment:
It's not a Gradle way. Properties have get() that could return null.
So we could use not nullable Properties without value as default